src/main/resources/schemas/iac_daily_list.json

```diff
@@ -327,6 +327,63 @@
                                                         "pattern": "^(?!.*<[^>]+>).*$"
                                                       }
                                                     }
+                                                  },
+                                                  "organisationDetails": {
+                                                    "title": "Organisation Details",
+                                                    "type": "object",
+                                                    "properties": {
+                                                      "organisationName": {
+                                                        "title": "Organisation Name",
+                                                        "type": "string",
+                                                        "pattern": "^(?!.*<[^>]+>).*$"
+                                                      },
+                                                      "organisationAddress": {
```

Comment:
Only organisationName needs to be here, so address / offence can be removed. Same for ET schemas too